doc/source/tutorial/optimize.rst

```diff
@@ -916,6 +1006,50 @@ We'll now plot all found minima on a heatmap of the function::
    :alt: "This X-Y plot is a heatmap with the Z value denoted with the lowest points as black and the highest values as white. The image resembles a chess board rotated 45 degrees but heavily smoothed. A red dot is located at many of the minima on the grid resulting from the SHGO optimizer. SHGO shows the global minima as a red X in the top right. A local minima found with dual annealing is a white circle marker in the top left. A different local minima found with basinhopping is a yellow marker in the top center. The code is plotting the differential evolution result as a cyan circle, but it is not visible on the plot. At a glance it's not clear which of these valleys is the true global minima."
    :include-source: 0
 
+Comparison of Global Optimizers
```

Comment:
```suggestion
Global optimization solver comparison
```
To match "Local minimization solver comparison"
But if that's the only thing, I won't change it here.